### DISCUSSION TITLE
Disable/enable espanso in search bar commands

### DISCUSSION THREAD

**Discussion by saravanabalagi** [espanso/espanso#1505] [Feature requests and Ideas]
Option + Spacebar opens the search bar and > shows commands. It would be nice if disable and enable espanso command in the list of commands, it is currently not available.

This is especially useful when using espanso with [tray icon off](https://github.com/espanso/espanso/issues/95).

**Comment by AucaCoyan** [comment]
Let me see if I understand correctly, do you mean typing `>` not triggering the espanso options?
That's a common experience in a couple of programs (vs code, windows terminal, obsidian, and more).

Why would we need to disable it? What benefit would give to the user or the software?

**Comment by smeech** [comment]
I can see that an item to disable Espanso in:
![image](https://github.com/user-attachments/assets/7f9a6af6-96d3-4510-bbfd-348e58de1d57)
would be useful as an alternative to hitting ones `toggle_key:` or using this mythical tray icon. :smile: 
_Enable_ isn't going to work if Espanso isn't running!
